spec/features/system_admin/dead_jobs/viewing_dead_jobs_spec.rb

```diff
@@ -26,6 +26,10 @@
     ]
   end
 
+  around do |example|
+    Timecop.freeze(Time.zone.now.change(hour: 12, minute: 0)) { example.run }
+  end
```

Comment:
Added a tiny flakey test fix here. Re-running at a different time of day would work but this is hopefully a little more permanent.